the-scheduled-lambda/typescript/lib/the-scheduled-lambda-stack.ts

```diff
@@ -19,9 +19,9 @@ export class TheScheduledLambdaStack extends Stack {
 
     // Create the Lambda function we want to run on a schedule
     const scheduledLambda = new Function(this, 'scheduledLambda', {
-        runtime: Runtime.NODEJS_12_X,
-        code: Code.fromAsset('lambda-fns/scheduled-lambda'),
```

Comment:
I moved the function to lambda-fns instead lambda-fns/scheduled-lambda. There is only one function, so there is no need to use many folders.